moveit_core/collision_detection/src/allvalid/collision_robot_allvalid.cpp

```diff
@@ -51,7 +51,7 @@ void collision_detection::CollisionRobotAllValid::checkSelfCollision(const Colli
 {
   res.collision = false;
   if (req.verbose)
-    logInform("Using AllValid collision detection. No collision checking is performed.");
+    CONSOLE_BRIDGE_logInform("Using AllValid collision detection. No collision checking is performed.");
```

Comment:
why is this change necessary? we weren't getting deprecated / compatibility warnings were we? I see [this file](http://docs.ros.org/api/class_loader/html/console__bridge__compatibility_8h.html) but would like to understand the backstory. This change reduces code readability.

Reply:
We were getting deprecated warnings, actually. The exact warning that clang gives is this:
```
/home/brycew/Desktop/moveit_ws/src/moveit/moveit_core/collision_detection/src/allvalid/collision_robot_allvalid.cpp:54:5: warning: 'log_deprecated' is deprecated [clang-diagnostic-deprecated-declarations]
    logInform("Using AllValid collision detection. No collision checking is performed.");
    ^
/usr/include/console_bridge/console.h:96:19: note: expanded from macro 'logInform'
  console_bridge::log_deprecated(__FILE__, __LINE__, console_bridge::CONSOLE_BRIDGE_LOG_INFO,  fmt, ##__VA_ARGS__)
                  ^
/usr/include/console_bridge/console.h:202:54: note: 'log_deprecated' has been explicitly marked deprecated here
CONSOLE_BRIDGE_DEPRECATED CONSOLE_BRIDGE_DLLAPI void log_deprecated(const char *file, int line, LogLevel level, const char* m, ...);
```

And looks like [this file](https://github.com/ros/console_bridge/blob/master/include/console_bridge/console.h#L81) does deprecate the current version we're using unfortunately. 

And from [this pull request](https://github.com/ros/console_bridge/pull/48/files), it looks like `logInform` et al. are about to be removed soon.